src/composables/useRuleForm.ts

```diff
@@ -77,6 +78,7 @@ export function useRuleForm(policies: Ref<Policy[]>, rule?: Ref<Rule | undefined
   const sourcePort = ref('')
   const destinationAddress = ref('')
   const destinationPort = ref('')
+  const sticky_toggle = ref(false)
```

Comment:
Let's use the same style of other variables, e.g. `stickyToggle` or simply `sticky`